todo-app/src/components/AddTodo.js

```diff
@@ -1,18 +1,47 @@
 import React from 'react';
+    var validTodo = "form-control";
 
 class AddTodo extends React.Component {
-    state = {  }
-    render() { 
+   
+    constructor(props) {
+        super(props);
+        this.state = {  "task": '',
+                        "complete":"no",
+                     };    
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+        
+    handleChange(event) {
+        validTodo = (event.target.value === '') ? "form-control border border-danger" : "form-control";
+        this.setState({task: event.target.value});
+        console.log('',validTodo );
+        
+    }        
+    handleSubmit(event) {    
+        event.preventDefault();    
+        console.log('', this.state.task);
+        if(this.state.task === ''){
+            alert('plesae enter a todo');

```

Comment:
instead of alert use toast message library `https://github.com/fkhadra/react-toastify`  and use it as a service.